src/views/Events/Events.module.scss

```diff
@@ -0,0 +1,26 @@
+.searchPageTitle {
+  color: var(--mui-palette-secondary-main);
+}
+.event_filter_icon {
+  color: var(--mui-palette-neutral-500);
+  font-size: 20;
+}
+.event_filter_list_icon {
+  color: var(--mui-palette-neutral-500);
+  font-size: 20;
+}
+.eventIcon {
+  color: var(--mui-palette-neutral-500);
+  font-size: 20;
+}
```

Comment:
is there a reason we can't do something along the lines of:

```
.event {
    font-size: 20;
    &_filter_icon{
      color: var(--mui-palette-neutral-500);
    }
 ...    
}
```?

Reply:
Nope, not that I know of, I can change that if we want.